src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -5226,6 +5226,45 @@ CompareShardPlacements(const void *leftElement, const void *rightElement)
 
 
 /*
+ * CompareShardPlacementsByNode compares two shard placements by their nodename
+ * and nodeport.
+ */
+int
+CompareShardPlacementsByNode(const void *leftElement, const void *rightElement)
+{
+	const ShardPlacement *leftPlacement = *((const ShardPlacement **) leftElement);
```

Comment:
Why did we add this function to physical planner? I think we should move it to where we use it as a static function.